Applications/Utils/MeshEdit/RemoveGhostData.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * \file
+ *
+ * @copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/LICENSE.txt
+ */
+
+#include <tclap/CmdLine.h>
+
+#include "Applications/ApplicationsLib/LogogSetup.h"
+#include "InfoLib/GitInfo.h"
+
+#include "MeshLib/IO/writeMeshToFile.h"
+#include "vtkCleanUnstructuredGrid.h"
```

Comment:
This is now a system include
```suggestion
#include <vtkCleanUnstructuredGrid.h>
```